PermissionsUtilsUI/src/main/res/layout/activity_app_diagnosis.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@mohamed-garci The layout has no toolbar. When activity is created, no toolbar will appear